packages/onboarding/src/step-container-v2/components/TopBar/TopBar.tsx

```diff
@@ -7,9 +8,10 @@ interface TopBarProps {
 	leftElement?: ReactNode;
 	rightElement?: ReactNode;
 	logo?: ReactNode;
+	className?: string;
 }
 
-export const TopBar = ( { leftElement, rightElement, logo }: TopBarProps ) => {
+export const TopBar = ( { leftElement, rightElement, logo, className }: TopBarProps ) => {
```

Comment:
Please, let's not add overrides to `StepContainerV2`'s components.

Reply:
My guess is that the logo component you're injecting has some absolute positioning, causing it to go out of the box, rendering an empty TopBar and therefore the weird padding issue you're seeing. Can you check that's the case?

Reply:
Yeah you were right, thanks for your input. This logo is used as is with it's own class in the other pages as well, so I reverted the TopBar changes 🙏 81b05cf